lib/graph3d/Graph3d.js

```diff
@@ -321,20 +326,23 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
 }
 
 
+/**
+ * Get the min/max values for the passed data column.
+ *
+ * Result is returned as a Range instance with min/max members
+ * properly set.
+ */
 Graph3d.prototype.getColumnRange = function(data,column) {
-  var minMax;
+  var range  = new Range();
 
+  // Adjust the range so that it covers all values in the
+  // passed data elements.
```

Comment:
Because:

- Like this it fits within the opening brace
- I'm totally paranoid about the 80 column limit

In other words, this is purely subjective and personal. No other reason.